lib/gobstones/compilation_mode.rb

```diff
@@ -48,7 +48,7 @@ def default_code
         private
 
         def blockly_code?(content)
-          Nokogiri::XML(content).errors.empty?
+          content.strip.start_with? '<xml'
```

Comment:
I'm guessing this one is for consistency with _other implementations_ and to avoid adding overhead to non-blocks operations.
I was about to raise a warning about malformed xml and this change, but the previous version treated it like non_blockly_code which is worse in retrospective.

I agree with this change :+1: Maybe in the future we could add some validation and better error report for this case, but if it happens, it is a symptom of bigger problem, sooo :man_shrugging:

Reply:
I'd like to add another concern: empty responses. Previous implementation was considering the `""` solution as XML/blockly, but it isn't. Please notice that I added a test for it.

Reply:
Please look at this commit: https://github.com/mumuki/mumuki-gobstones-runner/pull/198/commits/5e9d6bf3996704fe1ae119f96ad40dcf0fd1fffd